doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -188,6 +163,72 @@ preamble similar to the one below, indicating that this is an executable file:
 
 You don't need to edit this preamble, as it is autogenerated.
 
+Links, formatting and images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When converting a notebook to a MyST Markdown file, you may need to adjust some
+of the content to match the MyST Markdown syntax.
+
+- **External links:** MyST Markdown uses the standard Markdown syntax for links.
+  For example, ``[link text](https://www.example.com)``. If you have a link in a
+  notebook cell, you can convert it to the MyST Markdown format by enclosing the
+  link text in square brackets and the URL in parentheses.
+- **Internal links:** For internal cross-references such as links to SciPy
+  classes or functions, as well as intersphinx links, you can use the following
+  syntax: ``{role}`link text <reference>```, where ``role`` can be ``ref``,
+  ``class``, ``func`` or any other role you would use with rST. For example, to
+  link to the ``scipy.stats.bartlett`` function, use
+  ``{func}`scipy.stats.bartlett```.
+- **Formatting:** MyST Markdown supports standard Markdown formatting, such as
+  bold text, italic text, and code blocks. For example, to make text bold, you
+  can use double asterisks: ``**bold text**``. To make text monospace/formatted
+  as code, you can use *single_ backticks: ```code```.
+- **Images:** If your notebook contains images, you can include them in the MyST
+  Markdown file by using the following syntax:
+  ``![alt text](path/to/image.png)``. Images are usually stored in the same
+  folder as the MyST Markdown file, but you can also use relative paths to
+  reference images in other folders. Note that outputs to executed cells should
+  not be included in version control, as they will be automatically generated
```

Comment:
How might that happen? I seem to recall a setting that prevented this, and I think in your instructions you mentioned that the output should be cleared before saving the notebook.

Reply:
Exactly - maybe I should rephrase. What I mean is you should NOT manually add any of the output images as they will be automatically generated when the docs are built. When you run `python dev.py doc` all notebooks with a jupytext preamble get executed, so the outputs are generated and appended to the notebook.

Reply:
I guess what I'm getting at is whether the user needs to double check whether output is sneaking into the MyST markdown notebooks. It's mentioned twice, suggesting that the user needs to be vigilant, but in my experience with Jupyter Lab, I don't manually need to clear the notebook *or* double check; output of code cells is never saved. If this is because of a particular setting I have or a feature of Jupyter Lab that is uncommon among notebook editors, I can understand the need to mention it. However, if is something that should be transparent to the user if they have followed the rest of the instructions, I would document it differently. (E.g. I would maybe add a warning to the relevant instruction that deviations could result in output being saved to the markdown notebooks.)

Reply:
My fear is if I add an item tellling people how to add images, they will assume they have to add images from the output as well, which is not the case. But I now added that the instructions are for "static images" so maybe that's enough and I can remove that sentence.